LICENSE

```diff
@@ -0,0 +1 @@
+This work is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/3.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
```

Comment:
Leading single space looks unintended to me. Is it intended?

Reply:
Nope, weird. My branch did not have that extra space.
Anyway, reformatted now.